Sources/Xcodeproj/pbxproj().swift

```diff
@@ -94,7 +94,7 @@ public func pbxproj(srcroot: String, projectRoot: String, xcodeprojPath: String,
         print("            buildPhases = (\(module.compilePhaseReference), \(module.linkPhaseReference));")
         print("            buildRules = ();")
         print("            dependencies = (\(module.nativeTargetDependencies));")
-        print("            name = \(module.name);")
+        print("            name = \(module.c99name);")
```

Comment:
I don't think this is what we want, did this fix a problem you encountered on one of the tests?